workbench/src/renderer/server_requests.js

```diff
@@ -208,7 +208,6 @@ export async function saveToPython(payload) {
  */
 export async function archiveDatastack(payload) {
   const { port, id } = await getPortAndID(payload.model_id);
-  payload.model_id = id;

```

Comment:
For these two endpoints, use the original id in the format `model_id@version` rather than stripping it down to the `model_id`.